### PR TITLE
AT_BLOCK_NUMBER env var to export at specific block

### PR DESCRIPTION
**Pull request by mnaamani**

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default async function createApi() {
   const WS_URL = process.env.WS_URL || 'ws://127.0.0.1:9944'
 
   // explicitely state what RPC we are connecting to
-  console.log('Connecting to RPC at: ' + WS_URL)
+  console.error('Connecting to RPC at: ' + WS_URL)
```
Comment (ondratra) [inline src/api.ts:11]: This is only an informational message that was introduced to have certainty about what node I'm connecting to. Why is there a change to `.error()`?

**Review by ondratra — COMMENTED** [review]
Looks good.

File: src/api.ts (hunk 1 above)
Comment (mnaamani) [reply src/api.ts:11]: so it logs to stderr and not stdout. This way when you pipe the output of the script to a .json file you don't need to edit it and remove this line :)

Comment (ondratra) [reply src/api.ts:11]: Ok, does make sense

**Review by ondratra — APPROVED** [review]

**Pull request merged**